Documentation/Index.rst

```diff
@@ -13,31 +13,11 @@ This documentation is generated from PHP Source code of TYPO3 CMS.
 
 .. note::
 
-   This reference is created and updated manually.
-
-Current plans are to automate the rendering and to update the source code to
-provide a fully useful reference of all provided ViewHelpers by TYPO3 CMS.
-
-Right now this might look ugly in some places, e.g. ``= EXAMPLE =`` and rendered
-HTML markup like ``<code>``. This will be polished in the future.  The current
-state still is much better then nothing.
+   This reference is one time generated for Version 8.7.
 
 Also notice that package names are not 1:1 Composer packages. ``cms-`` is
 stripped.
 
-Contribute
```

Comment:
Is it intentional to remove the information aobut contribution?

Reply:
Yes, as contribution for v8.7 is completely different.